src/main/java/com/javadiscord/javabot/service/help/HelpChannelManager.java

```diff
@@ -224,7 +248,7 @@ private void unreserveChannelByOwner(TextChannel channel, User owner, Interactio
 				if (!entry.getKey().getUser().equals(owner)) potentialHelpers.add(entry.getKey());
 			}
 			if (potentialHelpers.isEmpty()) {
-				Responses.info(interaction.getHook(), "Channel Unreserved", "Your channel has been unreserved.").queue();
+				Responses.info(interaction.getHook(), null, "Your channel has been unreserved.").queue();
```

Comment:
It would be better if you made overloaded methods in `Responses` which did not take a title in the first place, and add explanation in the javadoc for those methods which do accept a nullable title.